src/components/game-board/squares/GameBoardSquares.js

```diff
@@ -8,7 +8,7 @@ import XPieceSVG from '../../game-pieces/x-svg/XPieceSVG';
 import OPieceSVG from '../../game-pieces/o-svg/OPieceSVG';
 
 // TODO: Move much of this to a common library.
-const pieces = [EmptyPieceSVG, XPieceSVG, OPieceSVG];
+const pieces = { 0: EmptyPieceSVG, 1: XPieceSVG, 4: OPieceSVG };
```

Comment:
Still need to centralize this mapping somewhere. That is:
Empty = 0
X = 1
O = 4

There is some math to that, but the idea goes back to my original TTT game in Python/Django